### PR TITLE
Improve documentation for Metrics API

### PR DESCRIPTION
**Pull request by santileira**
## Checklist before you pull:

[] Please ensure that there is no customer-identifying information in text or images.
[] Please ensure there is no consumer PII in text or images.
[] If you renamed any pages then perhaps you likely want to add a redirect from old to new URL so that we don't end up with broken links. You can do this by adding a pair of to/from values in `docusaurus.config.js` under `@docusaurus/plugin-client-redirects`

### CODE REVIEW

File: docs/embrace-api/supported_metrics_and_queries.md [hunk 1]
```diff
@@ -44,76 +45,62 @@ However, we strongly recommend transitioning to the new metrics to ensure a cons
...
 
-You can pull data for one, multiple, or all of your organization's apps in a single query.
+### Device Models
+There are over 40,000 unique device models on the Android operating system.  The bottom 39,000 models account for ~30% of data typically.  Aside from being expensive to store this many unique values, it is also unwieldy to visualize or review!
+
+<img src={require('@site/static/images/embrace-api/device_other.png').default} alt="Chart showing data by device ranking" />
+
+Currently, we roll together these long-tail device models into an "other" value.
+
+## Metrics Availability
```
Comment (santileira) [inline docs/embrace-api/supported_metrics_and_queries.md:99]: @scottieb3 I added this graph to explain when the metrics will be available. Could you read it and check if it makes sense and it is clear enough?

File: docs/embrace-api/supported_metrics_and_queries.md [hunk 2]
```diff
@@ -44,76 +45,62 @@ However, we strongly recommend transitioning to the new metrics to ensure a cons
 | crashes_total_deprecated                     | Number of crashes                                         | app_version, os_version               | five_minute, hourly, daily |
 | sessions_by_device_total_deprecated          | Number of sessions grouped by device model                | app_version, device_model, os_version | hourly, daily              |
 | sessions_total_deprecated                    | Number of sessions                                        | app_version, os_version               | five_minute, hourly, daily |
-| users_deprecated                             | Number of unique users                                    | app_version, os_version               | hourly, daily              |
-
-
-
-### Dimension reduction - "Other"
```
Comment (santileira) [inline docs/embrace-api/supported_metrics_and_queries.md:51]: This is common across all the metrics, I moved it out of standard metrics section

File: docs/embrace-api/supported_metrics_and_queries.md [hunk 3]
```diff
@@ -4,14 +4,15 @@ description: Learn about the metrics and queries supported by the Embrace API
 sidebar_position: 3
 ---
 
-# Metrics API Supported Metrics
+# Supported Metrics
 
-## Standard Metrics
+The following metrics are supported. Metrics with the suffix "_total" are gauges.
```
Comment (santileira) [inline docs/embrace-api/supported_metrics_and_queries.md:9]: This is common across all the metrics, I moved it out of standard metrics section

File: docs/embrace-api/supported_metrics_and_queries.md [hunk 4]
```diff
@@ -44,76 +45,62 @@ However, we strongly recommend transitioning to the new metrics to ensure a cons
...
-#### Device Models
-There are over 40,000 unique device models on the Android operating system.  The bottom 39,000 models account for ~30% of data typically.  Aside from being expensive to store this many unique values, it is also unwieldy to visualize or review!
-
-<img src={require('@site/static/images/embrace-api/device_other.png').default} alt="Chart showing data by device ranking" />
-
-Currently, we roll together these long-tail device models into an "other" value.
+## Custom Metrics
 
-### Sample Queries
+Refer to this [documentation](/custom-metrics-api/supported_metrics.md) to know the supported custom metrics.
```
Comment (santileira) [inline docs/embrace-api/supported_metrics_and_queries.md:52]: This was wrong. It was not explaining the custom metrics that we support.

File: docs/embrace-api/grafana_integrations.md [hunk 5]
```diff
@@ -14,11 +14,15 @@ sidebar_position: 1
 ## Setting Up Embrace as a Data Source
 
 Follow these steps to add Prometheus as a data source in the Grafana dashboard:
-1. Click the gear icon to go to the Configurations page.
```
Comment (santileira) [inline docs/embrace-api/grafana_integrations.md:17]: This was wrong. So I decided to fix it.

Comment (scottieb3) [reply docs/embrace-api/grafana_integrations.md:17]: I'm struggling with Grafana != Grafana Cloud.  But this was too clunky.

File: docs/embrace-api/supported_metrics_and_queries.md (hunk 1 above)
Comment (scottieb3) [reply docs/embrace-api/supported_metrics_and_queries.md:99]: Thank you

**Review by scottieb3 — APPROVED** [review]

File: docs/embrace-api/grafana_integrations.md (hunk 5 above)
Comment (santileira) [reply docs/embrace-api/grafana_integrations.md:17]: @scottieb3 do you prefer Grafana or Grafana Cloud? I can do the change to have consistency.

**Review by pablomatiasgomez — APPROVED** [review]

File: docs/embrace-api/metrics-migration.md [hunk 6]
```diff
@@ -6,7 +6,7 @@ sidebar_position: 5
 
 # Metrics migration (2023-10-23)
 
-Embrace is updating its beta Metrics API to support interoperability.  This means all metrics, both the pre-built or "Standard" and Custom Metrics, can be combined in any analyses you plan on doing.  Sum across time-ranges and other dimensions, compare one Metric with another, or use multiple metrics to compute relevant rates.
+Embrace is updating its beta Metrics API to support interoperability.  This means all metrics, both the pre-built or Standard and Custom Metrics, can be combined in any analyses you plan on doing.  Sum across time-ranges and other dimensions, compare one Metric with another, or use multiple metrics to compute relevant rates.
```
Comment (davidlawrencer) [inline docs/embrace-api/metrics-migration.md:9]: > This means all metrics, both the pre-built or Standard and Custom Metrics, can be combined in any analyses you plan on doing.

Can you make this sentence "This means all metrics, both the pre-built Standard Metrics and your Custom Metrics, can be combined in any analyses you plan on doing.

**Pull request merged**